### ISSUE TITLE
src/Hakyll/Web/Paginate.hs:40:17: No instance for (Show (PageNumber -> Identifier))

### ISSUE THREAD

**Issue by juhp** [jaspervdj/hakyll#422]
hakyll-4.8.2.0 failed to build in Stackage Nightly:

```
src/Hakyll/Web/Paginate.hs:40:17:
    No instance for (Show (PageNumber -> Identifier))
      (maybe you haven't applied enough arguments to a function?)
      arising from the second field of ‘Paginate’
        (type ‘PageNumber -> Identifier’)
    Possible fix:
      use a standalone 'deriving instance' declaration,
        so you can specify the instance context yourself
    When deriving the instance for (Show Paginate)
```

**Comment by bacher09** [comment]
I just wanted to write about it. It is same issue as #420.

**Comment by juhp** [comment]
For now I added a constraint on regex-tdfa in Stackage Nightly.

**Comment by neongreen** [comment]
See https://github.com/jaspervdj/hakyll/pull/421.

**Comment by jaspervdj** [comment]
Fixed in `hakyll-4.8.3.0`.
